src/event.js

```diff
@@ -11,16 +11,19 @@ jQuery.event.fix = function( originalEvent ) {
 		props = jQuery.event.props;
 
 	if ( props.length ) {
-		migrateWarn( "jQuery.event.props are deprecated and removed" );
+		migrateWarn( "jQuery.event.props are deprecated and removed: " + props.join() );
```

Comment:
This seemed useful info, at least it's something to search for and the name gives a clue as to what might have added it.